skrf/time.py

```diff
@@ -127,100 +123,132 @@ def indexes(y, thres=0.3, min_dist=1):
                 rem[sl] = True
                 rem[peak] = False
 
-        peaks = np.arange(y.size)[~rem]
+        peaks = npy.arange(y.size)[~rem]
 
     return peaks
 
-def find_n_peaks(x,n, thres=.9, **kw):
-    '''
-    Find a given number of peaks in a signal
-    '''
-    
-    for dummy  in range(10):
-        
-        idx = indexes(x, **kw)
+def find_n_peaks(x: npy.ndarray, n: int, thres: float = 0.9, **kwargs):
```

Comment:
Don't forget the type hint for the return value. Probably `-> List[int]`

Reply:
ok. Often the return value is NumberLike (scalar, list or numpy aray), which I remember is problematic (not here).